src/main/java/io/nitric/api/document/Query.java

```diff
@@ -350,9 +354,9 @@
      * @return the Query operation
      */
     public Query<T> where(String operand, String operator, Boolean value) {
-        Contracts.requireNonBlank(operand, "operand");
-        Contracts.requireNonBlank(operator, "operator");
-        Contracts.requireNonNull(value, "value");
+        Contracts.requireNonBlank(operand, OPERAND);
+        Contracts.requireNonBlank(operator, OPERATOR);
+        Contracts.requireNonNull(value, VALUE);
 
         expressions.add(new Expression(operand, operator, value));
```

Comment:
Would we be able to de-duplicate these methods with a private unified method add new expressions, e.g.:

```java
private Query<T> whereObj(String operand, String operator, Object value) {
  Contracts.requireNonBlank(operand, OPERAND);
  Contracts.requireNonBlank(operator, OPERATOR);
  Contracts.requireNonNull(value, VALUE);

  expressions.add(new Expression(operand, operator, value));
  
  return this;
}

public Query<T> where(String operand, String operator, Bool value) {
  return this.whereObj(operand, operator, value);
}
```

Reply:
Yep, will give that a go.